CHANGELOG.md

```diff
@@ -1,9 +1,675 @@
+- Remove BugReporting extraData log ([16eccf304e](https://github.com/facebook/react-native/commit/16eccf304e607684a7bc9be77ac9e59f6894dea8) by [@gaearon](https://github.com/gaearon))
+- Update integration snapshots ([2c7a907b08](https://github.com/facebook/react-native/commit/2c7a907b08fabcd774e037c8f8e5a9d56c1c57e9) by [@grabbou](https://github.com/grabbou))
+- Add UI thread assertions and annotations to ReactRootView ([c82e27030c](https://github.com/facebook/react-native/commit/c82e27030c20cfe12ab3f0f27022649fe48af5ae) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Add asserts and annotations for FabricUIManager UI-thread methods ([7c20dc65d9](https://github.com/facebook/react-native/commit/7c20dc65d9092b21a2e0d85b641c8319c059357e) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Attempt to collate Remove/Delete mounting items ([27fca36a9a](https://github.com/facebook/react-native/commit/27fca36a9a177c8b232201b9673a933e15844710) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Collect more information on Debug T48643168 ([fb90f64d9c](https://github.com/facebook/react-native/commit/fb90f64d9c5ead26e243a9a7bae80ba5487d1cff) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Fix generated props CPP default values ([cafbf67d0d](https://github.com/facebook/react-native/commit/cafbf67d0dc8b3674c82b86c9e9724ee4363890e) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Fix memory leak in experiment: JNI leak ([4b59360a59](https://github.com/facebook/react-native/commit/4b59360a59a8eb83bd6878559ef2b7a5392a8b10) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Give a more explicit message about why `number` isn't supported ([11dc847ec9](https://github.com/facebook/react-native/commit/11dc847ec9c491f5a8eeff1300bca02e040f5889) by [@JoshuaGross](https://github.com/JoshuaGross))
+- In Binding.cpp, retain reference to reactNativeConfig ([d7d848e824](https://github.com/facebook/react-native/commit/d7d848e8241ae430472ded5d428cacffe1e8bec7) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Remove hack in C++ that prevents any real TextInputs from being rendered ([219e1972d6](https://github.com/facebook/react-native/commit/219e1972d6f4ceeac4ffc41f8facc4205da78546) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Slightly more aggressive FabricUIManager teardown ([fa1c2be387](https://github.com/facebook/react-native/commit/fa1c2be387a95d90d8fcdaabb88b1c4c61e0097b) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Support for UpdatePaddingMountItem ([44bfc4be44](https://github.com/facebook/react-native/commit/44bfc4be447680096f01fd00a1344c2cd398a981) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Unbreak master ([a1ee1d7698](https://github.com/facebook/react-native/commit/a1ee1d7698081e7882d2fc5f08d848f93623f1c4) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Work around crashing redbox ([43f042eb7d](https://github.com/facebook/react-native/commit/43f042eb7db0f
```

Comment:
Should this change be under the `#### iOS Unknown` category?

Reply:
The `Unknown` category isn’t actually part of the normal changelog - it’s items that haven’t been categorized yet. Ideally, all the `Unknown` items should all be gone when the changelog is finalized, so we’re not moving anything else into there